src/custom/state/price/reducer.ts

```diff
@@ -23,9 +24,11 @@ export interface PriceInformation {
 }
 
 export interface QuoteInformationObject extends FeeQuoteParams {
-  fee: FeeInformation
-  price: PriceInformation
+  fee?: FeeInformation
+  price?: PriceInformation
+  error?: ApiErrorCodes
   lastCheck: number
+  // TODO: remove
```

Comment:
Is this comming in a new PR where we kill the `feeExceedsPrice`?

Reply:
yes